whitelabelco/whitelabel-components
# Setup storybook addons

Resolves: Issue #35

## Description
This PR:
- Sets up a11y-addon for storybook
- Reverts storybook version to 6.5

## To Test :mag: 
- Go to Buttons component
- Verify that the accessibility tab is present, should be showing one error for text in the button


## Checklist :white_check_mark:
- [ ] I have re-assigned the Github ticket and moved the ticket on the project board.
- [ ] If I have added an ENV var or other secrets, I have updated Dashlane and the ReadMe (and any other applicable docs)
